arangod/RocksDBEngine/RocksDBEdgeIndex.cpp

```diff
@@ -259,18 +259,28 @@ bool RocksDBEdgeIndexIterator::nextCovering(DocumentCallback const& cb, size_t l
   }
 #endif
 
+  VPackBuilder coveringBuilder;
```

Comment:
Can we do better here, and reuse an existing temporary Builder from the transaction, by using `TransactionBuilderLeaser` here?